README.md

```diff
@@ -204,27 +206,38 @@ regular HDFS with the exception of providing a different file system, e.g.:
 import java.net.URI
 import org.apache.hadoop.fs.FileSystem
 import org.apache.spark.sql.SparkSession
+import za.co.absa.atum.core.Atum
 
 val spark = SparkSession
       .builder()
       .appName("Example Spark Job")
       .getOrCreate()
 
+implicit val atum = Atum // using basic Atum without extensions
```

Comment:
I don't think you can with the current design, because you don't want regular Atum implicit object to be in scope when AtumSdkS3 should be used and the latter inherits from the former. 

So at least I made another implicit wrapper so just import 
```scala
import za.co.absa.atum.AtumImplicits._
import za.co.absa.atum.AtumImplicitsCore._ 
```
to work with base atum without having to manually define your implicit atum object and

```
import za.co.absa.atum.AtumImplicits._
import za.co.absa.atum.AtumImplicitsSdkS3._
import za.co.absa.atum.AtumImplicitsSdkS3Core._ 
```
when working with the extended version. Again, no need to define your own Atum object here, too.

Not perfect, but closer to your idea.

What I would need here is something like `import implicit` to actually group existing implicit import into one wrapper which I am not aware that would exist. (I'd be glad if you proved me wrong :D)